.github/workflows/azure_ci_check.yml

```diff
@@ -0,0 +1,74 @@
+name: Azure CI
+
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize, edited ]
+  issue_comment:
+    types: [ created, edited ]
+
+jobs:
+  check-azure-ci-report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get last commit hash
+        id: last_commit
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const pr = context.payload.pull_request;
+            const lastCommitHash = pr.head.sha;
+            console.log(`Last commit hash: ${lastCommitHash}`);
+            // Set the output variable to be used in subsequent step
+            core.setOutput("last_commit_hash", lastCommitHash);
+
+      - name: Check Azure CI report in PR comment
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const lastCommitHash = '${{ steps.last_commit.outputs.last_commit_hash }}'
```

Comment:
nit: remove the spaces in the brackets: ${{ steps.last_commit.outputs.last_commit_hash }}

Reply:
Actually GitHub docs suggest this pattern with spaces before and after the variable: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows